db/migrate/20151015183729_update_miq_database_default_repo_name.rb

```diff
@@ -0,0 +1,33 @@
+class UpdateMiqDatabaseDefaultRepoName < ActiveRecord::Migration
+  class MiqDatabase < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    has_many :authentications, :as => :resource
```

Comment:
Is there a good way to be sure we are not loading the entire model in the process of the migration?  Or will it be enough to just not use the `db.authentications` query?